integreat_cms/cms/views/media/media_context_mixin.py

```diff
@@ -62,6 +62,8 @@ def get_context_data(self, **kwargs):
                 "label_file_uploaded": _("File uploaded on:"),
                 "label_file_modified": _("File modified on:"),
                 "label_directory_created": _("Directory created on:"),
+                "label_directory_is_hidden": _("Directory is hidden:"),
+                "label_file_is_hidden": _("File is hidden:"),
```

Comment:
```suggestion
                "label_file_is_hidden": _("Hide this file in region media library:"),
```

I thik this is more self-explaining.